app/views/layouts/application.html.erb

```diff
@@ -59,6 +59,11 @@
                 <%= link_to 'New Group', new_group_path, class: "dropdown-item" %>
               </div>
             </li>
+            <li class="nav-item">
+              <%if user_signed_in?%>
+                <%= link_to 'Log Out', signout_path, class: "nav-link text-light" %>
```

Comment:
@kostik700015 Remember to try to keep pull requests to one single issue. This is important, but not related to the Google Map API, and could have been added in a separate pull request.

Reply:
Yes. I tested that. I just accidently found bug and fixed it while worked on gmaps api adding. Bad news - we cant do working gmaps api unless we get  an api key with billing. So its like work but, doesn't because it has no key.